src/controllers/request.controller.js

```diff
@@ -222,4 +226,64 @@ export default class RequestController {
       return errorResponse(res, badRequest, error.message);
     }
   }
+
+  /**
+   * @param {Request} req Node/express request
+   * @param {Response} res Node/express response
+   * @returns {Object} Response object with success message
+   * @description approves a trip request
+   */
+  static async approveTripRequest(req, res) {
+    const { tripRequestId } = req.params;
+    const status = ACCEPTED;
+    const current = req.tripRequestCurrentStatus;
+    if (current === status) {
+      return errorResponse(res, conflict, customMessages.tripRequestAlreadyApproved);
+    }
+    if (current === REJECTED) {
+      return errorResponse(res, forbidden, customMessages.tripRequestAlreadyRejected);
+    }
+    const handledBy = req.sessionUser.id;
+    await updateTripRequestStatus(tripRequestId, status, handledBy);
+    return successResponse(res, ok, customMessages.requestApprovalSuccess, undefined, undefined);
+  }
+
+  /**
+   * @param {Request} req Node/express request
+   * @param {Response} res Node/express response
+   * @returns {Object} Response object with success message
+   * @description rejects a trip request
+   */
+  static async rejectTripRequest(req, res) {
+    const { tripRequestId } = req.params;
+    const status = REJECTED;
+    if (status === req.tripRequestCurrentStatus) {
+      return errorResponse(res, conflict, customMessages.tripRequestAlreadyRejected);
+    }
+    const handledBy = req.sessionUser.id;
+    await updateTripRequestStatus(tripRequestId, status, handledBy);
+    return successResponse(res, ok, customMessages.requestRejectionSuccess, undefined, undefined);
+  }
+
+  /**
+   * @param {Request} req Node/express request
+   * @param {Response} res Node/express response
+   * @returns {Object} Response object with success or error message
+   * @description assigns trip request to another manager
+   */
+  static async assignTripRequest(req, res) {
+    const { tripRequestId } = req.params;
+    const handledBy = req.newManager.id;
```

Comment:
give your variables better names. handledBy may be obvious to you, but for others it may take some time before we figure out what it means.